src/test/java/com/epam/HelloUnitTest.java

```diff
@@ -1,10 +1,12 @@
 package com.epam;
 
+import lombok.extern.slf4j.Slf4j;
 import org.junit.jupiter.api.Test;
-
-public class HelloUnitTest {
+@Slf4j
+class HelloUnitTest {
```

Comment:
Please rename the test class and test method to make it more meaningful. For example Slf4jTest and test loggingInfoLevel()

Reply:
Done